content/ch-quantum-hardware/ac-Stark-shift.ipynb

```diff
@@ -46,10 +46,12 @@
     "# Importing standard Qiskit libraries and configuring account\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        qc_stark_circs = [qc_stark.assign_parameters({amplitude: amp , drive_freq: f, control_freq: f - 100 * MHz}, inplace=False) for f in freqs]
</pre><p>This code looks cool! :100:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1455/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Just being picky, but single-letter variables are not good for code readability.</p>

Reply:
Thank you for your review.

I will change variable name from f to freq.